core/micrometer/src/test/java/org/eclipse/dataspaceconnector/metrics/micrometer/MicrometerExtensionIntegrationTest.java

```diff
@@ -17,22 +17,21 @@
 import okhttp3.OkHttpClient;
 import okhttp3.Request;
 import okhttp3.Response;
-import org.eclipse.dataspaceconnector.azure.testfixtures.annotations.OpenTelemetryIntegrationTest;
+import org.eclipse.dataspaceconnector.common.annotations.OpenTelemetryIntegrationTest;
 import org.eclipse.dataspaceconnector.junit.launcher.EdcExtension;
-import org.junit.jupiter.api.BeforeAll;
+import org.eclipse.dataspaceconnector.junit.launcher.OpenTelemetryExtension;
 import org.junit.jupiter.api.BeforeEach;
 import org.junit.jupiter.api.Test;
 import org.junit.jupiter.api.extension.ExtendWith;
 
 import java.io.IOException;
-import java.lang.management.ManagementFactory;
-import java.lang.management.RuntimeMXBean;
 import java.util.Map;
 
 import static org.assertj.core.api.Assertions.assertThat;
 import static org.eclipse.dataspaceconnector.common.testfixtures.TestUtils.getFreePort;
 
 @OpenTelemetryIntegrationTest
+@ExtendWith(OpenTelemetryExtension.class)
```

Comment:
nit: @ExtendWith takes array so both extension can be added in one.

No need to change just highlighting. Thanks

Reply:
Thanks, actually I moved the @ExtendWith(OpenTelemetryExtension.class) to the @OpenTelemetryIntegrationTest tag.